modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/MultiSelectItemListBuilder.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib.util;
+
+import com.liferay.petra.function.UnsafeConsumer;
+import com.liferay.petra.function.UnsafeSupplier;
+
+/**
+ * @author Carlos Lancha
+ */
+public class MultiSelectItemListBuilder {
+
+	public static MultiSelectItemListWrapper add(MultiSelectItem multiSelectItem) {
```

Comment:
Hey @carloslancha I have some questions about this builder, just so it's not such a mystery to me.

1. Why do we define all these `add` methods?
2. What are `unsafeSupplier` and `unsafeConsumer`?
  - I assume they're there for security? To prevent some weird values being `put` inside the `MultiSelectItem`?
3. Why do we, almost, duplicate all of the `add` methods up here and below inside the `final class`?
  - I notice they're different, the bottom ones are like definitions where the top ones are like usages/overrides

Reply:
@carloslancha I'm getting 17 instances of this error when deploying `clay-sample-web` with your changes:
```bash
error: cannot find symbol multiSelectItem.setValue("3");
                          ^
symbol:   variable multiSelectItem
```

Reply:
I didn't test it, I just copied the same pattern from LabelItemListBuilder. I'll take a look

Reply:
Pushed a commit fixing it, some variables had a wrong name.

Reply:
Thanks!